IaC/AKS/README.md

```diff
@@ -0,0 +1,373 @@
+# Deployment Walk Through
+
+## Background
+The following instructions allow the deployment of NGSA application in AKS with Istio and Keda.
+
+### Azure Components in Use
+
+- Azure Kubernetes Service
+  - Istio ServiceMesh
+  - Keda
+  - Prometheus
+  - Kiali
+  - Grafana 
+- Azure Cosmos DB
+- Application Insights
+
+### Prerequisites
+
+- Azure subscription with permissions to create:
+  - Resource Groups, Service Principals, Cosmos DB, AKS, Azure Monitor
+- Bash shell (tested on Mac, Ubuntu, Windows with WSL2)
+  - Will not work in Cloud Shell or WSL1
+- Azure CLI ([download](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest))
+- Docker CLI ([download](https://docs.docker.com/install/))
+- Visual Studio Code (optional) ([download](https://code.visualstudio.com/download))
+- kubectl (install by using `sudo az aks install-cli`)
+- Helm v3 ([Install Instructions](https://helm.sh/docs/intro/install/))
+
+### Setup
+
+Fork this repo and clone to your local machine
+
+```bash
+
+cd $HOME
+
+git clone https://github.com/retaildevcrews/ngsa
+
+```
+
+Change into the base directory of the repo
+
+```bash
+
+cd ngsa
+
+export REPO_ROOT=$(pwd)
+
+```
+
+#### Login to Azure and select subscription
+
+```bash
+
+az login
+
+# show your Azure accounts
+az account list -o table
+
+# select the Azure account
+az account set -s {subscription name or Id}
+
+```
+
+This walkthrough will create resource groups, a Cosmos DB instance, and an Azure Kubernetes Service (AKS) cluster.
+
+#### Choose a unique DNS name
+
+```bash
+
+# this will be the prefix for all resources
+# do not include punctuation - only use a-z and 0-9
+# must be at least 5 characters long
+# must start with a-z (only lowercase)
+export He_Name=[your unique name]
```

Comment:
Since `He` indicates helium, let's prefix the env variables with something for NGSA

Reply:
agree